lib/THC/generic/THCTensorCopy.cu

```diff
@@ -31,16 +31,27 @@ THCTensor_(copyIgnoringOverlaps)(THCState* state, THCTensor* dst, THCTensor* src
   }
 
 IMPLEMENT_THC_CUDA_TENSOR_COPY(Byte, Byte)
+
+#ifndef TH_GENERIC_NO_CHAR
```

Comment:
Thanks for catching this: yes it seems I was compiling w/o those macros defined - apologize for the confusion.